addons/point_of_sale/static/src/app/sound/sound_service.js

```diff
@@ -1,15 +1,23 @@
 import { SoundEffects } from "@mail/core/common/sound_effects_service";
-import { patch } from "@web/core/utils/patch";
+import { registry } from "@web/core/registry";
 
-patch(SoundEffects.prototype, {
-    play() {
+class PoSSoundEffects extends SoundEffects {
+    constructor(env) {
+        super(env);
         this.soundEffects = {
             ...this.soundEffects,
             error: { path: "/point_of_sale/static/src/sounds/error" },
             bell: { path: "/point_of_sale/static/src/sounds/bell" },
             notification: { path: "/point_of_sale/static/src/sounds/notification" },
             beep: { path: "/point_of_sale/static/src/sounds/beep" },
         };
-        super.play(...arguments);
+    }
+}
+
+export const soundService = {
+    start(env) {
+        return new PoSSoundEffects(env);
     },
-});
+};
```

Comment:
i also thought about this. The advantage is that we don't have to replace `sound` with `mail.sound_effects` everywhere, but this doesn't really seem like an "advantage". Why would we basically alias the name? It's nicer to immediately see what service you are actually using, not having the name be obfuscated by the alias. 

What do you think?